packages/blocks/src/page-block/default/default-page-block.ts

```diff
@@ -443,11 +443,6 @@ export class DefaultPageBlockComponent
       if (flag) window.addEventListener('keydown', this._handleNativeKeydown);
       else window.removeEventListener('keydown', this._handleNativeKeydown);
     });
-
-    this.model.page.slots.blockUpdated.on(() => {
```

Comment:
I think we still need this to make sure programmatic adding new blocks will work as expected, as stated in PR summary in https://github.com/toeverything/blocksuite/pull/1520

Reply:
The `page.blockUpdated` triggers at every nested block update, which may be too coarse-grained. So I tried to use `model.childrenUpdated` event instead, which only triggers at shallow children update.